src/seraphis_impl/serialization_demo_types.h

```diff
@@ -344,14 +350,17 @@ struct ser_SpMembershipProofV1_PARTIAL final
 /// serializable SpTxSupplementV1
 struct ser_SpTxSupplementV1 final
 {
-    /// xKe: enote ephemeral pubkeys for outputs
+    /// D_e[]: enote ephemeral pubkeys for outputs
     std::vector<crypto::x25519_pubkey> output_enote_ephemeral_pubkeys;
     /// tx memo
     std::vector<unsigned char> tx_extra;
+    /// npbits
+    std::uint8_t num_primary_view_tag_bits;
```

Comment:
Yes, since I think it would be a good idea to be able to adjust `npbits` according to relay rules. Otherwise, it would have to be inferred from some version number, and updating that version number would result in a hard fork.